README.md

```diff
@@ -5,7 +5,7 @@
 ## Build tools
 
 This project uses Gulp to run a number of automated tasks, including one that grabs our source files (HTML, Sass, JS)
-and compiles them into one single HTML file that has everything inlined and can be deployed to auth0. 
+and compiles them into one single HTML file that has everything inlined and can be deployed to auth0.
```

Comment:
Not your text, but want to flag this is outdated. 

Suggested alternative for this whole paragraph:

> This project uses Gulp to build our static assets (CSS, JS, images, fonts) into a folder that can be deployed to a CDN. It also builds a HTML file that points to the right versions of those assets.

Reply:
Good call, I'll fix that.